src/main/java/spring/bookstore/service/BookServiceImpl.java

```diff
@@ -1,9 +1,9 @@
-package spring.book_store.service;
+package spring.bookstore.service;
 
-import org.springframework.stereotype.Service;
-import spring.book_store.model.Book;
-import spring.book_store.repository.BookRepository;
 import java.util.List;
+import org.springframework.stereotype.Service;
+import spring.bookstore.model.Book;
+import spring.bookstore.repository.BookRepository;
 
 @Service
```

Comment:
Why this service is already in master branch? 
```suggestion
@Service
@RequiredArgsConstructor
```

remove constructor